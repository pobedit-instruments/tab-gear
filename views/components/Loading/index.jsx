import React, { Component, PropTypes } from 'react';

import Text from '../Text';

class Loading extends Component {
	render () {
		return <Text> { chrome.i18n.getMessage('wait') }Â </Text>;
	}
}

Loading.propTypes = {};

export default Loading;
