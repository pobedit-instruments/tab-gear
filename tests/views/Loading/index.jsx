import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow, mount, render } from 'enzyme';

import Loading from '../../../views/components/Loading';

test('Loading view', () => {
	let component = renderer.create(<Loading />);
	let view = component.toJSON();

	let actual = expect(view);

	actual.toMatchSnapshot();
});
