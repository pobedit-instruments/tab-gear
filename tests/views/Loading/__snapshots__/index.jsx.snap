exports[`test Loading view 1`] = `
<div
  className="tg-box">
   
   
  You should to be patient…
   
   
</div>
`;
